doc/userguide/rules/smb-keywords.rst

```diff
@@ -0,0 +1,16 @@
+SMB Keywords
+==============
+
+SMB keywords used in both SMB1 and SMB2 protocols.
+
+smb.filename
```

Comment:
Could you please explain the difference with generic `file.name` keyword in the doc ?